src/components/ChannelList/__tests__/ChannelList.test.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import axios from 'axios';
-import { cleanup, render, waitFor } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { getNodeText } from '@testing-library/dom';
```

Comment:
If https://github.com/GetStream/stream-chat-react/pull/236/files#r430243870 works we can get rid of this.